Fix font color from HandleClass

rcla opened the pull request:
Gets the font color property from HandleClass and updates it.

Before with Bulma (theme-light):
![BulmaBefore](https://github.com/pascaldragon/Pas2JS_Widget/assets/45245617/3a97a89b-949a-4e92-a749-3d7a0a32c338)

After with Bulma (theme-light):
![BulmaAfter](https://github.com/pascaldragon/Pas2JS_Widget/assets/45245617/e16f913f-54ea-4de8-a4c9-2f6030367f7a)


Before with Bulma (theme-dark):
![BulmaDarkBe](https://github.com/pascaldragon/Pas2JS_Widget/assets/45245617/11003870-6651-4fab-bd2e-f97845a37c0d)


Afert with Bulma (theme-dark):
![BulmaDarkAf](https://github.com/pascaldragon/Pas2JS_Widget/assets/45245617/1fb9ffb6-935e-45fb-82e2-f1dc4e2ebe11)


Before with Bootstrap:
![BootstraBef](https://github.com/pascaldragon/Pas2JS_Widget/assets/45245617/a153073f-ef95-4112-916f-8883eb82595f)

After with Bootstrap:
![BootstrapAf](https://github.com/pascaldragon/Pas2JS_Widget/assets/45245617/06ef569b-ad1b-479a-858b-37cc7e2fc4af)


To apply Bulma's light and dark themes, it is necessary to assign the TWForm color in clNone and in the html file:
```html
<html lang="en" class="theme-dark">
```
Or
```html
<html lang="en" class="theme-light">
```

rcla committed:
Fix font color from HandleClass

Gets the font color property from HandleClass and updates it.

bbrtj commented:
I think it's better to replace line
```
Style.SetProperty('color', JSColor(FFont.Color));
```
with this:
```
if (FFont.Color in [clDefault, clNone]) then
  Style.RemoveProperty('color')
else
  Style.SetProperty('color', JSColor(FFont.Color));
```
This is pretty much the same as what's done to background color. It removes default styles from html, so that css styles for class work.
Edit: It actually needs a couple more changes. I may open a separate pull request for that.

rcla commented:
@bbrtj 
That solution wouldn't let me change the font color when I have HandleClass<>''

I currently have it working like this:
[Fix Font Color](https://github.com/rcla/Pas2JS_Widget/blob/ff2e8abd720e58ababf7546518543066301be74e/widgets/controls.pas#L1612)

rcla commented:
Anyway, I made this Pull Request almost nine months ago and I have changed many things.

Not receiving any response, I have made my own fork and there I have changed several details that I have found and added other features.

So it's not worth continuing to leave it open. I'll close it.
